app/views/shared/_footer_app.html.haml

```diff
@@ -11,3 +11,6 @@
           %a{href:'https://docs.google.com/a/umn.edu/forms/d/e/1FAIpQLSf6OHrj6YIcn8yNe6sWgQJ_iJfElDZgozUCSdATzePwLqul5Q/viewform'} Contact Project Team
         %li
           %a{href:'http://www.btaa.org/homepage'} Big Ten Academic Alliance Homepage
+        %li
+          = link_to image_tag('twitter-logo.png') + ' Follow @geobtaa', 'https://twitter.com/geobtaa'
```

Comment:
Please add an `:alt` argument to the image_tag:
https://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-image_tag

All images need an alternative text value to be minimally accessible:
https://webaim.org/techniques/alttext/

Reply:
The alt text for Twitter logo image was added for the latest commit. Thanks for your reminder.